### PR TITLE
Add alternative installation method

### PR DESCRIPTION
**Pull request by leni8ec**
Scoop is a command-line installer for Windows, mainly for portable applications.

### CODE REVIEW

**Comment by Kyome22** [comment]
@leni8ec 
I haven't done anything to Scoop, but is it ready to install RunCat?

**Comment by b4imetu** [comment]
> @leni8ec I haven't done anything to Scoop, but is it ready to install RunCat?

Scoop is always ready and has updated to Runcat 2.0 now
https://github.com/ScoopInstaller/Extras/commits/master/bucket/runcat.json

**Comment by Kyome22** [comment]
@b4imetu 
Does Scoop only support x64?
https://github.com/ScoopInstaller/Extras/blob/f1afdc598d93398d2e4a5629b5fac17c0ad2a30d/bucket/runcat.json#L9

**Comment by b4imetu** [comment]
@Kyome22 
Scoop doesn't create packages, it's just a community-maintained Windows package management tool (like Chocolatey). Scoop provides the appropriate software distribution based on the packages/releases in your repository.
If you provide 32-bit software, the maintainers of the scoop (community) can add 32-bit architecture downloads for your software.

File: README.md [hunk 1]
```diff
@@ -23,6 +23,9 @@
 
 **Access to the "Releases" page and download the RunCat.exe.**
 
+- Or install via [Scoop](https://github.com/ScoopInstaller/Scoop):
+`scoop install runcat`
+
```
Comment (Kyome22) [inline README.md]: @leni8ec 
Please specify that this way is for the x64 version.

**Review by Kyome22 — APPROVED** [review]
LGTM

**Pull request merged**